Resources/public/js/views/ez-contenteditview.js

```diff
@@ -313,7 +352,16 @@ YUI.add('ez-contenteditview', function (Y) {
                         version: this.get('version')
                     });
                 }
-            }
+            },
+
+            /**
+             * The language code in which the content is edited.
+             *
+             * @attribute languageCode
+             * @type {String}
+             * @required
+             */
+            languageCode: {},
```

Comment:
for consistency this attribute should also be set on the first content edit display before the editor uses the change link (you have to change `ContentEditViewService._getViewParameters` to also return its `languageCode` attribute + add/change the corresponding test). AFAICT, that also means you'll have to change the `languageCodeChange` event handler to check for the `active` attribute before trying to update the DOM.